model/facade/action/GetPersonalSummaryByLoginDateAction.php

```diff
@@ -194,12 +177,11 @@ protected function doExecute() {
         if( $this->currentJourney ) {
             $extraGoalHoursSet = 0;
             if ( $this->currentUserGoal ) {
```

Comment:
This if is not required now; you deleted other conditions but probably forgot this one :)